Lib/fontParts/base/normalizers.py

```diff
@@ -100,10 +100,6 @@ def normalizeKerningKey(value):
             raise FontPartsError("Kerning key items must be strings, not %s." % type(v).__name__)
         if len(v) < 1:
             raise FontPartsError("Kerning key items must be one character long")
-    if value[0].startswith("@") and not value[0].startswith("@public.kern1."):
```

Comment:
would this be better:
`if value[0].startswith("public.") and not value[0].startswith("public.kern1."):

That would catch using non-kerning groups in kerning, or the wrong kerning group being used